lib/iser.c

```diff
@@ -108,16 +103,27 @@ iscsi_iser_service(struct iscsi_context *iscsi, int revents)
 	int ret = 0;
 	struct iser_conn *iser_conn = iscsi->opaque;
 
+	if (iscsi->pending_reconnect) {
+		if (time(NULL) >= iscsi->next_reconnect) {
```

Comment:
Please use clock_gettime(CLOCK_MONOTONIC, ...) instead of a function like time() that returns wall clock time

Reply:
These lines are consistent with those in iscsi_tcp_service() (socket.c:903), and next_reconnect is calulated by ```iscsi->next_reconnect = time(NULL) + backoff;``` in iscsi_reconnect_cb() (connect.c:300).
It might be better to modify them together in a separate commit if it matters.

Reply:
That sounds fine to me.